go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/redhat-developer/service-binding-operator
 
-go 1.13
+go 1.14.3
```

Comment:
we should not be so precise here, 1.14 should work.

Reply:
@pedjak 

Hi, Just to understand, Is there a technical reason why we don't provide the patch number?

Reply:
this here is about go compiler version, not the bugfix version. It is signal to compiler what language features will be available:

https://blog.ksub.org/bytes/2019/09/15/go.mods-go-directive/

Reply:
Oh thanks for explanation!